BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/BaseBrick.java

```diff
@@ -164,6 +165,26 @@ public void setStickyScrollMode(@StickyScrollMode int stickyScrollMode) {
     }
 
     /**
+     * Whether the brick should take up the width or height of the fragment
```

Comment:
can full width not be calculated somewhere else using the span size of the brick and span size of the brickdatamanager?